CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-03-31
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* Force migrating all Image Cleaner clusters to addon v2.
+
+### Release Notes
+* Features:
+  * Created ZRS disk by default in built-in storage classes on a multi-zone AKS cluster
+
+* Component Updates: 
+  * Bump csi driver to v1.4.2.
+  * Bump cloud-controller-manager 1.29.3, 1.28.8, 1.27.16, 1.26.22
+  * Bump Windows image versions to 2024.3B Windows 2019: March 12, 2024—KB5035849 (OS Build 17763.5576). Version: 17763.5576.240316. Windows 2022: March 12, 2024—KB5035857 (OS Build 20348.2340). Gen1 Version: 20348.2340.240316. Gen2 Version: 20348.2340.240316.
```

Comment:
vhd notes are missing. we need to add those and reference them similar to the previous release notes

Reply:
This was actually already announced in last release notes so I removed.